src/backend/ai-reviewer-api/src/test/java/ca/bc/gov/open/jag/aireviewerapi/document/validators/documentValidator/ValidateExtractedDocumentTest.java

```diff
@@ -80,20 +78,6 @@ public void withValidDocumentExecuteWithoutThrowing() throws JsonProcessingExcep
 
         answers.add(documentAnswerField);
 
-        DiligenAnswerField plaintiffAnswerField = DiligenAnswerField.builder()
-                .id(Keys.ANSWER_PLAINTIFF_ID)
```

Comment:
You can also remove the import for this class.  It's now unused.

Reply:
Thanks thought I caught all the unused imports